### ISSUE TITLE
unable to see the test failures I caused

### ISSUE THREAD

**Issue by bukzor-sentryio** [terraform-google-modules/terraform-google-kubernetes-engine#1487]
### TL;DR

When authoring a contribution, a CI test failed, and I'm unable debug or fix it due to lack of permissions.

### Expected behavior

Click on the "view details" link and see details of the test failure.

* my pull request https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/pull/1486
* CI failure: https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/pull/1486/checks?check_run_id=9830240675
* test failure details: https://console.cloud.google.com/cloud-build/builds/17729caa-2bf9-4acc-82ca-eb2e6e9f4bd7?project=2843445864

### Observed behavior

You do not have sufficient permissions to view this page
There was an error while loading [/cloud-build/builds/17729caa-2bf9-4acc-82ca-eb2e6e9f4bd7?project=2843445864](https://console.cloud.google.com/cloud-build/builds/17729caa-2bf9-4acc-82ca-eb2e6e9f4bd7?project=2843445864).

The required permissions are:
Project  2843445864

cloudbuild.builds.get

### Terraform Configuration

```hcl
n/a
```

### Terraform Version

```sh
n/a
```

### Additional information

Please grant `cloudbuild.builds.get` in project `2843445864` to the general public.
